spk/ser2net/PLIST

```diff
@@ -0,0 +1,29 @@
+bin:sbin/ser2net
```

Comment:
this whole file can be removed, since it is not used.
PLIST files are mostly used for cross builds only. They define the target files of a cross package to be copied into the staging folder when a spk target is built.